test/Microsoft.AspNetCore.Authentication.Test/MicrosoftAccountTests.cs

```diff
@@ -1,242 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved. See License.txt in the project root for license information.
-
-using System;
```

Comment:
??

Reply:
I was just making a half-baked commit to my repo. I didn't realize that it would become part of the pull request :) I can't get it to build properly on command line, VS Code, or VS...different error each place, so this is not ready.